src/NotificationTargetContract.php

```diff
@@ -180,6 +192,8 @@ public function getTags()
             'contract.type'         => _n('Type', 'Types', 1),
             'contract.entity'       => Entity::getTypeName(1),
             'contract.states'       => __('Status'),
+            'contract.endtime'      => __('Contract expired since the'),
+            'contract.noticetime'   => __('Contract with notice since the'),
```

Comment:
Thsi value will be available even before the contract expiration date, in this case, the contract will not yet be expired.

```suggestion
            'contract.endtime'      => __('Contract expiration date'),
            'contract.noticetime'   => __('Contract notice date'),
```

Reply:
@cedric-anne if we change here, we must change also other existing strings

Reply:
`contract.time` value is only defined when the contract is already expired or the notice period has already been reached.

https://github.com/glpi-project/glpi/blob/f1d7e39171583c89a070c67ec5c6f2aaf3ab450f/src/NotificationTargetContract.php#L103-L116

Reply:
Thank you @cedric-anne , "Contract expiration date" and "Contract notice date" are surely better labels. I used the other ones because they were already available & translated.  I commit your suggestion.